CondFormats/L1TObjects/interface/L1GtDefinitions.h

```diff
@@ -136,7 +138,8 @@ enum L1GtConditionCategory {
     CondHfBitCounts,
     CondHfRingEtSums,
     CondBptx,
-    CondExternal
+    CondExternal,
```

Comment:
@rekovic 
Are these extended enums used in the persistent data?  I mean, if the possible enum's values are stored in the DB, the previous releases won't be able to read the new data.